packages/experimental/Button/src/useButton.ts

```diff
@@ -0,0 +1,23 @@
+import { useAsPressable, useKeyCallback, useViewCommandFocus } from '@fluentui-react-native/interactive-hooks';
+import { ButtonProps, ButtonState } from './Button.types';
+
+export const useButton = (props: ButtonProps): ButtonState => {
```

Comment:
I would also decouple this from the Button package, if we:

- make the input props and returned props ViewProps. This will help with setting to the internal nodes as well.
- don't use the ButtonState object, just return a tuple of [ViewProps, PressableState]
- then this could also be put in interactive hooks

Reply:
I like this idea but I'm not sure it's quite as simple as you're suggesting if we want to get the typings right, so I'm going to punt on this for now.